src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$calendar-day-color: #eee;
+$calendar-lenght: 31;
+$calendar-day-height: 100px;
+$calendar-day-width: 100px;
+$calendar-day-font-size: 30px;
+$calendar-padding: 10px;
+$calendar-gap: 1px;
+$days: mon 1, tue 2, wed 3, thu 4, fri 5, sat 6, sun 7;
```

Comment:
The variable `$days` should be named to more accurately reflect its purpose. Consider renaming it to something like `$weekdays-start-index` to make it clear that it represents the start index of each weekday.